src/app.js

```diff
@@ -0,0 +1,48 @@
+import * as functions from './utils';
+
+export let typeSelect,
+    vehicleSelect,
+    modelSelect,
+    preco = document.createElement('h1');
+
+export const url = ['http://fipeapi.appspot.com/api/1/carros'];
+
+(function () {
```

Comment:
Wrapper isn't necessary anymore using webpack and the ES6 modules you are using guarantees that it does not go to the global scope. You can try to access `url` that was declared outside the wrapper and see that it is not accessible.